bin/hummingbot_quickstart.py

```diff
@@ -148,7 +148,13 @@ def main():
     else:
         secrets_manager = secrets_manager_cls(args.config_password)
 
-    asyncio.get_event_loop().run_until_complete(quick_start(args, secrets_manager))
+    try:
+        ev_loop: asyncio.AbstractEventLoop = asyncio.get_event_loop()
+    except Exception:
```

Comment:
Same with this exception

Reply:
At the least one of us should probably stick a comment in the code explaining the reason I guess